### ISSUE TITLE
Tasks

### ISSUE THREAD

**Issue by Leo-Lee92** [Leo-Lee92/drug-discovery-research#3]
## This repository lists up tasks that i should do

- [x] 각 SMILES (compound) 별 몇개의 FASTA (protein)이 연결되어 있는지 그리기
- [x] (가능하면) protein-compound network 그리기
- [ ] Evaluation metric 결정하기
- [ ] Transformer 구현하기
- [ ] token to word 한방에 할 수 있는 keras 또는 tensorflow 코드 짜기
- [ ] Domain 2 Domain Translation에 대해 공부하기
    - Domain 2 Domain 사례들 열거
    - Domain 2 Domain 에서 지식활용의 중요성 요약
- [ ] Knowledge Graph-based Convolution에 대해 공부하기
- [ ] 각종 search 알고리즘 기반 sampling 기법들 구현하기
    - Greedy search
    - Stochastic search
    - Beam (k = 4, 10, 20, ..) search
    - Neucleus search 
    - ...
- [ ] dd
